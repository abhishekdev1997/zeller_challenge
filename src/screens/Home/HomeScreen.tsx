import { View, StyleSheet } from "react-native"
import { Heading1 } from "../../components/Heading"

const HomeScreen = () => {
    return (
        <View style={styles.container}>
            <Heading1 title="Welcome to Home Screen" />
        </View>
    )
}

const styles = StyleSheet.create({
    container: {
        flex: 1,
        justifyContent: "center",
        alignItems: "center"
    }
})

export default HomeScreen