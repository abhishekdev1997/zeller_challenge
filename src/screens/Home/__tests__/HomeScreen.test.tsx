import 'react-native';
import React from 'react';
import HomeScreen from '../HomeScreen';
import { render } from "@testing-library/react-native";
jest.useFakeTimers()


it('homescreen renders correctly', () => {
    render(
        <HomeScreen />
    );
});
