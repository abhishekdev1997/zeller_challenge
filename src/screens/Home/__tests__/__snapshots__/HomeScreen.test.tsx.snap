// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`test homescreen snapshot homescreen renders correctly 1`] = `
<View
  style={
    {
      "alignItems": "center",
      "flex": 1,
      "justifyContent": "center",
    }
  }
>
  <Text
    style={
      [
        {
          "fontFamily": "System",
          "fontSize": 22,
          "fontWeight": "400",
          "letterSpacing": 0,
          "lineHeight": 28,
        },
        {
          "textAlign": "left",
        },
        {
          "color": "rgba(28, 27, 31, 1)",
          "writingDirection": "ltr",
        },
        {
          "fontWeight": "700",
        },
      ]
    }
  >
    Welcome to Home Screen
  </Text>
</View>
`;
